src/components/buy-together/services/front-buy-together.adapter.ts

```diff
@@ -194,4 +237,56 @@ export class FrontBuyTogetherAdapter {
       !checkHasBalance({ balance, isSellOutOfStock }) || !checkIsOutReleaseDate({ releaseDate })
     );
   }
+
+  private static getInstallmentsWithoutInterest(installments: PaymentInstallment[]) {
+    return installments.filter(installment => installment.markup <= 1);
+  }
+
+  private static getTotalInstallments(installments: PaymentInstallment[]) {
+    return installments;
+  }
+
+  private static adaptCreditCardPayment(
+    product: Product,
+    paymentsConfig: BuyTogetherPaymentsConfig,
+    actualPayment: Payment,
+  ): PaymentOption {
+    const installmentsNoInterest = this.getInstallmentsWithoutInterest(actualPayment.installments);
```

Comment:
```suggestion
    const installmentsNoInterest = actualPayment.installments;
```
Acho que não precisamos da função `getTotalInstallments` nesse caso

Reply:
feito